### PR TITLE
[SYCL] Fix true missing on Windows

### PR DESCRIPTION
**Pull request by alexbatashev**
Switch to `echo` instead of Unix-specific `true`

### CODE REVIEW

**Comment by steffenlarsen** [comment]
Will something similar be feasible for the test suite?

**Comment by alexbatashev** [comment]
> Will something similar be feasible for the test suite?

Working on that, however there's a bit more than just `true` for test suite. I believe some tests also use `cat` or `ls`.

**Comment by alexbatashev** [comment]
@steffenlarsen would you please try https://github.com/intel/llvm-test-suite/pull/383? Should fix your issue.

File: sycl/test/on-device/lit.cfg.py [hunk 1]
```diff
@@ -191,8 +191,8 @@ def getDeviceCount(device_type):
 
 found_at_least_one_device = False
 
-cpu_run_substitute = "true"
-cpu_run_on_linux_substitute = "true "
+cpu_run_substitute = "echo "
+cpu_run_on_linux_substitute = "echo "
```
Comment (s-kanaev) [inline sycl/test/on-device/lit.cfg.py:195]: Are arguments optional for `echo` on both windows and linux?

Comment (alexbatashev) [reply sycl/test/on-device/lit.cfg.py:195]: They're not, but a) I didn't find the case when we have no args b) both can accept more than 1 argument

**Review by smaslov-intel — APPROVED** [review]

**Pull request merged**